### DISCUSSION TITLE
WinMergeU - CLI - Is there any parameter that return a true/false is two files are identical or not

### DISCUSSION THREAD

**Discussion by pcakhilnadh** [WinMerge/winmerge#743] [Q&A] [answered]
Hi There,

I'm trying to compare two files using WinMerge CLI 

`WinMergeU text1.txt text2.txt -noninteractive -or output.html `

Is there any parameter that the cli returns a True/False value if the files are identical or not?

**Comment by sdottaka** [comment]
Unfortunately, the parameters you expected do not exist.

**Comment by sdottaka** [comment, marked as answer]
In the upcoming release 2.16.15, the /enableexitcode command line option will be added.

When this option is specified, the process exit code will be as follows

0: identical
1: different
2: error

~~~
start /wait ""  "c:\Program Files\WinMerge\WinMergeU.exe" /enableexitcode c:\dev\winmerge\src\Merge.cpp d:\dev\winmerge\src\Merge.cpp
echo %ERRORLEVEL%
~~~
